Add 2022-09-29 agenda and minutes (#74)
solid/notifications-panel: meetings/2022-09-29.md

* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
* Join queue to talk.

### Participation and Code of Conduct
* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
* If this is your first time, welcome! please introduce yourself.

### Scribes
* name

### Introductions

* name: text

---

## Topics

### WIP Implementation Feedback
* SC: We'll allocate some time for implementation feedback. Links to servers/applications and demos welcome.
* RG: https://syntropize.com/

> * RG: Please see the website and read preliminary documentation. I would like to show notification specific code to demonstrate my requirements.

Reviewer: ```suggestion
* RG: Please see the website and read preliminary documentation. I would like to show notification-specific code to demonstrate my requirements.
```

### Simplify flow diagrams
URL: https://github.com/solid/notifications/pull/108

>ACTION: AC to do minor updates.

### Discovery of Notification Subscription Metadata
URL: https://github.com/solid/notifications/issues/58

>ACTION: AC to PR updating the requirement.

### Add Solid Notification JSON-LD context
URL: https://github.com/solid/vocab/pull/62

> >SC: We need to follow up on this after resolving some issues in solid/notifications (linked in the review). It should be published this alongside the next release of the Notifications Protocol.

Reviewer: ```suggestion
>SC: We need to follow up on this after resolving some issues in solid/notifications (linked in the review). This should be published alongside the next release of the Notifications Protocol.
```

### Interchangeable use of notification channel type and subscription type
URL: https://github.com/solid/notifications/issues/71

### Define Notification Content
URL: https://github.com/solid/notifications/pull/60

* SC: Not critical for the next release but the underlying intent would be useful for the spec.
* SC: At the very least, right now it highlights what may be overspecifying, constraining, exposing information.
* SC: We should consider a simpler / minimal model for conformance.

> * SC: (I suggested a way that can beNotification Profiles

Reviewer: ```suggestion
* SC: (I suggested a way that can be Notification Profiles)
```

### Notification Profiles
URL: https://github.com/solid/notifications/issues/101

>Introduce the notion of Notification Profile as an extension of the core Notification Data Model.
>Specific Notification Profiles can then be independently published.

### Solid Notifications Protocol: Review notes
URL: https://github.com/solid/notifications/issues/76

>ACTION: AC to respond to design re performance of making the connection for live updates.

* SC: I wanted to respond to that but didn't get around to it.

### Too Slow: Please provide Updates-Via shortcut to secure web socket address
URL: https://github.com/solid/notifications/issues/110

### Cases where Client might want to update Subscriptions/Connections
URL: https://github.com/solid/notifications/issues/103

* SC: Action from https://github.com/solid/notifications-panel/blob/main/meetings/2022-08-18.md#an-alternative-websockets-design :

>ACTION: RG to create issue to note the cases in which client may want to update its subscriptions or connections.

> * SC: Before discussing that issue, does https://github.com/solid/notifications/issues/97 need a follow-up? Does 103 supersede 97? Close 97?

Reviewer: ```suggestion
* SC: Before discussing that issue, does https://github.com/solid/notifications/issues/97 need a follow-up? Does #103 supersede #97? Close #97?
```
